controllers/tasksRequests.js

```diff
@@ -0,0 +1,95 @@
+const { INTERNAL_SERVER_ERROR, SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+const taskRequestsModel = require("../models/taskRequests");
+
+const fetchTaskRequests = async (_, res) => {
+  try {
+    const taskRequests = await taskRequestsModel.fetchTaskRequests();
+
+    return res.status(200).json({
```

Comment:
Either use status or boom here to maintain consistency.

Reply:
`boom` is used for throwing error object, on a success response I am using `res.status`